com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/ProjectConfigHandler.java

```diff
@@ -0,0 +1,75 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Microsoft Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Microsoft Corporation - initial API and implementation
+ *******************************************************************************/
+
+package com.microsoft.java.debug.plugin.internal;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.jdt.core.IJavaProject;
+
+public class ProjectConfigHandler {
+    /**
+     * Return the java project settings.
+     * @param arguments - The first argument is the project name, the second argument indicate whether to get the inherited options.
+     * @return The map of current settings of all options(key type: String; value type: String)
+     */
```

Comment:
I think API should be flexible for multiple purpose yet concise. 

In here, every variable has its own API, which will bloat API number. We should keep the API design simple.

Reply:
addressed. Will change the api to check project settings. The api parameter is `className: String, projectName: String, inheritedOptions: boolean, expectedOptions: Map<String, String>`